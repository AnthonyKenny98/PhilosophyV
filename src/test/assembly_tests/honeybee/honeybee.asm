addi $x1, $x0, 0
addi $x2, $x0, 0
addi $x3, $x0, 0
addi $x4, $x0, 0
addi $x5, $x0, 0
addi $x6, $x0, 0
addi $x7, $x0, 0
addi $x8, $x0, 0
addi $x9, $x0, 0
addi $x10, $x0, 0
addi $x11, $x0, 0
addi $x12, $x0, 0
addi $x13, $x0, 0
addi $x14, $x0, 0
addi $x15, $x0, 0
addi $x16, $x0, 0
addi $x17, $x0, 0
addi $x18, $x0, 0
addi $x19, $x0, 0
addi $x20, $x0, 0
addi $x21, $x0, 0
addi $x22, $x0, 0
addi $x23, $x0, 0
addi $x24, $x0, 0
addi $x25, $x0, 0
addi $x26, $x0, 0
addi $x27, $x0, 0
addi $x28, $x0, 0
addi $x29, $x0, 0
addi $x30, $x0, 0
addi $x31, $x0, 0
#
#
# Begin Test
li.e $e0, 0.5
li.e $e1, 0.5
li.e $e2, 0.5
li.e $e3, 0.75
li.e $e4, 0.75
li.e $e5, 0.75
ecol $x1, $x2

