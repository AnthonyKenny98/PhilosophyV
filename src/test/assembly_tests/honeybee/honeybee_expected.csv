32
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0b00111111000000000000000000000000
0b00111111000000000000000000000000
0b00111111000000000000000000000000
0b00111111010000000000000000000000
0b00111111010000000000000000000000
0b00111111010000000000000000000000